README.md

```diff
@@ -157,6 +157,29 @@ You can parse these or pass them directly into a command to run tests, like so:
 pub run build_runner test $(pub run test_html_builder:browser_aggregate_tests --mode=args [--release])
 ```
 
+### Randomizing the browser aggregation file
+By default, the tests will be aggregating in a specific order and this will be maintained between
+each test run. In order to mimic the dart test runner's built in test shuffling capability a consumer can add
+this to their yaml file to randomly shuffle the test order.
+
+```yaml
+targets:
+  $default:
+    builders:
+      test_html_builder:
+        options:
+          ...
+          randomize_ordering_seed: random
+```
+
+If you notice a test failure due to the specific shuffling that occurred during that test
+run, update the test_shuffle_seed to be whatever was output in the test runners log.
```

Comment:
Oh this is a good point.. I didn't think about the fact that we'd want to be sure that the seed can be obtained from the logs for debugging purposes. Ok, maybe instead of logging the seed as info or warning, we could use `.shout(...)`? That should guarantee that it gets printed without it being misconstrued as a warning

Reply:
```suggestion
If you notice a test failure due to the specific shuffling that occurred during that test
run, update the `randomize_ordering_seed` option to be whatever was output in the test runners log.
```